src/css/components/menu/_menu-inline.scss

```diff
@@ -13,6 +13,7 @@
 // Hover state
 .video-js .vjs-menu-button-inline:hover,
 .video-js .vjs-menu-button-inline:focus,
+.video-js .vjs-menu-button-inline[aria-expanded="true"],
```

Comment:
I don't think that it really makes a difference in most browsers that we care about other than maybe browsers like IE8.
There isn't a class we can use right now and this attribute selector is also technically more semantically correct. We could toggle a class in addition if necessary.